rules/windows/discovery_whoami_command_activity.toml

```diff
@@ -17,31 +17,31 @@ false_positives = [
 ]
 from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
-language = "kuery"
+language = "eql"
 license = "Elastic License v2"
 name = "Whoami Process Activity"
```

Comment:
noisy, maybe limit it to `user.id :  ("S-1-5-18", "S-1-5-19", "S-1-5-20")`

Reply:
there is already another rule that look for whoami with system user.

Reply:
ignore the above if deprecating a rule will be handled in a different release/PR.